arcgis-ios-sdk-samples/Analysis/Statistical query (group and sort)/StatisticalQueryGroupAndSortViewController.swift

```diff
@@ -28,9 +28,23 @@ class StatisticalQueryGroupAndSortViewController: UIViewController, UITableViewD
     private var orderByFields = [AGSOrderBy]()
     private var selectedOrderByFields = [AGSOrderBy]()
     private var statisticDefinitions = [AGSStatisticDefinition]()
-    private var sectionHeaderTitles = ["1. Add Statistic Definitions", "2. Select Group By Fields", "3. Select Order By Fields"]
     private var statisticTypes = ["Average", "Count", "Maximum", "Minimum", "StandardDeviation", "Sum", "Variance"]
     
+    private enum Section: Int, CaseIterable {
+        case definitions, groupByFields, orderByFields
+        
+        var title: String {
```

Comment:
I know I've suggested this before, but since this is only useful in `titleForHeaderInSection`, I would suggest moving it there. It eliminates a level of indirection and keeps the interface of `Section` nice and concise.

PS: In general, it is helpful to keep the definition of value types as concise as possible, particular when they are subtypes. One way of doing that is to move computed properties into an extension.

Reply:
That's a fine point.